web-client/src/js/common/button/CustomButton.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import {div, Button} from 'react-bootstrap';
+//import TestDefinition from '../../view/testDefinition/TestDefinition';
+
+class CustomButton extends Component {
+  render() {
+    return (
+      <div>
+        <Button onClick={this.createTest} bsStyle="primary" bsSize="large" block>
+          Create Test
```

Comment:
Get the button text through props, so that we can use this button at multiple other places.